modules/ducktests/tests/ignitetest/tests/discovery_test.py

```diff
@@ -71,40 +72,65 @@ class DiscoveryTest(IgniteTest):
     """
     NUM_NODES = 7
 
-    FAILURE_DETECTION_TIMEOUT = 2000
+    FAILURE_DETECTION_TIMEOUT_TCP = 1000
+
+    FAILURE_DETECTION_TIMEOUT_ZK = 3000
 
     DATA_AMOUNT = 5_000_000
 
     WARMUP_DATA_AMOUNT = 10_000
 
+    NETFILTER_SAVED_SETTINGS = os.path.join(IgniteTest.TEMP_PATH_ROOT, "discovery_test", "netfilter.bak")
```

Comment:
This will not work in parallel execution (class variable), use `tempfile` module and instance variable

Reply:
Removed